### PR TITLE
Update node.md

### PR DESCRIPTION
**Pull request by Legends**
I wouldn't use `These` at the beginning, It would be viable if you'd mentioned node options before and now refer to them with `these`.

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/configuration/node.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ contributors:
   - EugeneHlushko
 ---
 
-These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
+The following node options below configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```
Comment (EugeneHlushko) [inline src/content/configuration/node.md]: I think we reference node officially as Node.js throughout the docs, lets do that here too if you dont mind
```suggestion
The following Node.js options below configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```

File: src/content/configuration/node.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ contributors:
   - EugeneHlushko
 ---
 
-These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
+The following Node.js options below configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```
Comment (EugeneHlushko) [inline src/content/configuration/node.md]:
```suggestion
The following Node.js options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```

Comment (Legends) [reply src/content/configuration/node.md]: LGTM

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**